azure/services/asogroups/groups.go

```diff
@@ -59,7 +59,7 @@ func (s *Service) Name() string {
 
 // Reconcile idempotently creates or updates a resource group.
 func (s *Service) Reconcile(ctx context.Context) error {
-	ctx, _, done := tele.StartSpanWithLogger(ctx, "groups.Service.Reconcile")
+	ctx, _, done := tele.StartSpanWithLogger(ctx, "asogroups.Service.Reconcile")
```

Comment:
this won't be possible yet since we're having to keep both packages side by side for now for UserAssigned identity but thoughts on removing the "aso" prefix everywhere in naming and going back to just "groups" as service name once we remove legacy groups?

Reply:
Yes, that's the plan.